tests/test_mutations.py

```diff
@@ -1570,8 +1572,11 @@ def cmp_mutation(a, b):
     # in order already). Kept mutations are given an id that is
     # their order in the initial tables, and new mutations have id -1.
     out = a.id * (not a.new) - b.id * (not b.new)
-    if out == 0:
+    either_unknown = tskit.util.is_unknown_time(a.time) or tskit.util.is_unknown_time(b.time)
```

Comment:
Maybe your'e in the middle of this, but:  because of the `unknown_time` flag, and sticking in the node.time, aren't these times never unknown?

Reply:
yes! this is a relic I need to get rid of